datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py

```diff
@@ -36,6 +36,16 @@
     'cryptography',
     'dnspython',
     'pymysql',  # https://github.com/DataDog/integrations-core/pull/12612
+    'protobuf',  # https://github.com/DataDog/integrations-core/pull/12653
```

Comment:
I dont think these comments are very informative. The link is this very same PR that is introducing the exception but it does not give any information as to why it's being added, nor does the PR description. Which test/integration does it break?

Reply:
I'm going to change the comment to indicate the integration being broken

Reply:
Done